app/src/main/res/layout/fragment_gameplay.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".GameplayFragment">
+
+
+    <TextView
+        android:id="@+id/tvScore"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginLeft="20dp"
+        android:layout_marginTop="20dp"
```

Comment:
We tend to store those dimensions inside `dimens.xml`, as they're usually standardized and storing them there makes it easier to change them for certain configurations (such as small or large screens).

We name them according to their usage and descriptive size, such as `margin_standard`, `margin_small`, `text_size_normal`, and so on.